x/finality/types/power_table.go

```diff
@@ -62,6 +62,22 @@ func (dc *VotingPowerDistCache) FindNewInactiveFinalityProviders(prevDc *VotingP
 		}
 	}
 
+	// Also check for finality providers that were active before but now
```

Comment:
what if we throw the event if the `fpDistinfo` is out of power 
```go
		if fpDistInfo.TotalBondedSat > 0 {
			newDc.AddFinalityProviderDistInfo(fpDistInfo)
		} else {
			// emit inactive event
```
and then remove `FindNewInactiveFinalityProviders`, I didn't double checked if that is enough but just an idea

Reply:
We'll need to add more logic to check if it was active before emitting the INACTIVE event. I believe will complicate and more this part of the code